docs/userGuide/templates.md

```diff
@@ -31,6 +31,8 @@ Name    | Template key | Description | Quick Deploy
 ----    | -------      | ----------- | ------------
 Default | `default`    | Default template if `--template` is unspecified. Includes core features such as site and page navigation for a more convenient quick start. | <a href="https://app.netlify.com/start/deploy?repository=https://github.com/MarkBind/init-typical-netlify"><img src="https://www.netlify.com/img/deploy/button.svg" /></a>
 Minimal | `minimal` | Minimalistic template that gets you started quickly. | <a href="https://app.netlify.com/start/deploy?repository=https://github.com/MarkBind/init-minimal-netlify"><img src="https://www.netlify.com/img/deploy/button.svg" /></a>
+Portfolio | `portfolio` | Portfolio template to quickly build your portfolio. | <a href=""><img src="https://www.netlify.com/img/deploy/button.svg" /></a>
```

Comment:
Hey Markbind team! I've been following this issue for quite a while but never got to see a preview of the Portfolio template. Maybe that is something you can consider adding to the documentation, perhaps a screenshot?

Reply:
Good point - it shouldn't be too hard to add a modal with image examples 👀 heres a short video i took when i was reviewing if you'd like to add your comments as well


https://github.com/MarkBind/markbind/assets/59471320/ffd799d2-1d0a-42be-b8a7-6b15493a1511

Reply:
Thanks! I took a short look during the lecture today too, and it looks clean! 

Not sure if this is possible on Markbind, but I think some of these expandable cards could be open by default? For example, the skills section, and maybe a few selected projects to emphasize the best ones.

I think the intro section ("My name is...") could be redesigned slightly if it is within the scope of this PR. For example, center the text, add a one-line intro, add call-to-action button(s) - for example, 'View Resume', or 'About Me', or the icon buttons for GitHub and LinkedIn can be added here

Just a few design suggestions, feel free to ignore

Reply:
Hi Charisma! Thank you for your feedback. I think I can definitely change the skills section so the panels are open by default. For the projects section, each project is shown on an individual card so they are already open by default. I looked at your website again and I agree with the "featured projects" and "other projects". Maybe the current big cards can be for featured projects, and I can add a grid of smaller cards underneath that to show other projects.

I think the "About me" section looks better with the text aligned to the left? It looks neater in my opinion. 😓 
Your comment on the github/linkedin/resume links makes sense! I can def move it to the "About me" section. 

Also sorry about not having a netlify link previously, I've since set it up here: https://portfolio-template-markbind.netlify.app/